packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -161,7 +161,7 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
         });
         keybindings.registerKeybinding({
             command: WorkspaceCommands.SAVE_AS.id,
-            keybinding: 'ctrl+shift+s',
+            keybinding: 'ctrlcmd+shift+s',
```

Comment:
Side note: Here, I don't believe it's useful to support the old keybinding on Mac OS.